src/main/java/frc/robot/commands/IntakeCommand.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot.commands;
```

Comment:
Existing files use 4 spaces for indentation, could we change this one to use 4 as well? Same goes for other files indented with 2 spaces.

Reply:
Done.